data/remnant missions.txt

```diff
@@ -1506,3 +1510,150 @@ mission "Remnant: Broken Jump Drive job"
 		dialog "The research team quickly offloads the broken jump drive and pays you <payment> before hurrying back to their lab."
 
 
+
+mission "Remnant: Face to Maw 1"
+	name "Talk to Plume about the Void Sprites"
+	description "Plume would be very interested in a report on what happened with the Void Sprites"
+	landing
+	to offer
+		has "Remnant: Return the Samples 2: done"
+	source
+		system "Nenia"
+	destination "Aventine"
+	on offer
+		conversation
+			`As the swirls of gas close over the <ship> you wonder idly what brought you back here. This time, there is no apologetic researcher asking you to risk your life for research, or to return eggs from where they were stolen. The last glimpse of the Archon drifting overhead as you descend below the cloud deck provides no answers; For now, it is nothing more than a silent, brooding presence in the skies of Nenia. The silent depths of an alien world fold over you, gradually erasing the rest of the galaxy from sight.`
+			`	As the <ship> levels off at the designated altitude, you strap yourself into the pressure suit. Stepping into the airlock, you lower the shields so you can exit onto the hull. Out on the surface, you admire the misty kaleidescope of colors filtering down through the cloud layers, playing over distant shapes moving in the mists. Before long several void sprites emerge, weaving and dancing among the clouds. They swirl for a few minutes around your cargo bay doors as if waiting for something then pulse briefly with flickers of light and then drop into the clouds below. The third void sprite flutters around the <ship>, poking at things and sliding over them. You quickly think to turn on your video recorder to catch the scene. The Void Sprite notices your brief movement, and turns towards you with a lunge. As it closes on you, your attention is drawn to what you suspect is its mouth: A zig-zagging crack that peels open like a zipper to reveal an opening that resembles an industrial grinder.`
```

Comment:
`void sprite` -> `Void Sprite`

`into the` -> `into a` (too many instances of "the" in the sentence)

`if waiting for something then pulse` -> `if waiting for something, then pulse`

`and then drop` -> `and drop` or `and finally drop` (too many instances of "then" in the sentence)

`The third void sprite` -> `another void sprite` (it wasn't previously established that there were two; "several" is deliberately ambiguous)

Reply:
Thanks. Everything fixed.